### DISCUSSION TITLE
Best way to have multiple yabai configs (profiles)

### DISCUSSION THREAD

**Discussion by carlosflorencio** [asmvik/yabai#834] [Q&A]
Hi,

I want to switch between yabai configurations with a custom script, I would then call this script via a skhd shortcut.

The logic I'm looking for should be something like this (this isn't working ofc):
```bash
#!/bin/bash

file="$(pwd)/currentMode"

if grep -Fxq "personal" "$file"
then
    # set custom configs for work profile
    yabai -m space 1 --label "1:slack"
    yabai -m space 4 --label "f1:outlook"

    echo "work" > "$file" # toggle profile
else
    # set custom configs for personal profile
    yabai -m space 1 --label "1:dev"
    yabai -m space 4 --label "f1:slack"

    echo "personal" > "$file"
fi

# reload yabai with the new configs
launchctl kickstart -k "gui/${UID}/homebrew.mxcl.yabai"
```

Is there a simple way to make this work? 

A more complex solution that should work is to make the script update the yabai config file with some inline replaces and trigger a reload, but I'd like to keep the config file clean.

Thanks!

**Comment by SxC97** [comment]
I think Yabai supports the config command `yabai -c path/to/config` so you can keep extra config files in a folder and set the following command to an SKHD shortcut `brew services stop yabai && brew services start yabai -c path/to/config`.

More information can be found in the [wiki](https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc#options).

**Reply by carlosflorencio** [reply]
Hi, thanks for answering.
I would like to avoid duplicating the whole config file. For the time being I'm using `sed` to comment/uncomment some lines on the config file and does the trick.
